libtsuba/src/AddProperties.h

```diff
@@ -33,7 +33,8 @@ AddProperties(
 
     auto add_result = add_fn(props);
     if (!add_result) {
-      return add_result.error();
+      return add_result.error().WithContext(
+          "adding \"{}\" property", properties.name);
```

Comment:
Not an endorsement but apparently there is some support for quoting strings.

Context:

https://github.com/fmtlib/fmt/issues/825

and suggested solution

https://en.cppreference.com/w/cpp/io/manip/quoted

e.g.,
```WithContext("adding {}", quoted(properties.name))```